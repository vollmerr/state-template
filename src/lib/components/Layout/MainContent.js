import React from 'react';

import LayoutBase from './LayoutBase';

const MainContent = (props) => (
  <LayoutBase baseClass={'main-content'} id={'main-content'} {...props} />
);

export default MainContent;
