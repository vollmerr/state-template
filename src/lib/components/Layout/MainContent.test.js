import React from 'react';
import { shallow } from 'enzyme';

import { MainContent as fromLib } from '../..';
import MainContent from './MainContent';

const props = {
  children: 'test-children',
  className: 'test-class',
};

let wrapper;
describe('MainContent', () => {
  beforeEach(() => {
    wrapper = shallow(<MainContent {...props} />);
  });

  it('should be exported as part of the library', () => {
    expect(fromLib).toEqual(MainContent);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
