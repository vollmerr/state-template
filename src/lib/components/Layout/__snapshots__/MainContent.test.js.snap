// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContent should render correctly 1`] = `
<LayoutBase
  baseClass="main-content"
  className="test-class"
  id="main-content"
  tag="div"
>
  test-children
</LayoutBase>
`;
